storage/simulation/cluster.go

```diff
@@ -223,45 +223,92 @@ func (c *Cluster) prepareActions() {
 
 // performActions performs a single action, if required, for each range.
 func (c *Cluster) performActions() {
-	// TODO(Bram): convert this to run a single action per store instead of a
-	// single action per range.
-	for _, rangeID := range c.rangeIDs {
-		r := c.ranges[rangeID]
-		nextAction, rebalance := r.getNextAction()
-		switch nextAction {
-		case storage.AllocatorAdd:
-			newStoreID, err := r.getAllocateTarget()
-			if err != nil {
-				fmt.Fprintf(c.actionWriter, "Error: %s\n", err)
-				continue
+	// Each range can only go through one action per epoch. If there is a
+	// conflict, the store will not perform an operation as this simulates two
```

Comment:
sorry, i wasn't being pedantic - i was asking which of the conflicting operations would actually go through. the one with the higher priority, i guess. i think this is reasonable clear now